Notebook.ipynb

```diff
@@ -321,7 +344,14 @@
    "id": "bf6a7369",
    "metadata": {},
    "source": [
-    "Here, we sum all the rewards coming from the events we created."
+    "Here, we sum all the rewards coming from the events we created.\n",
+    "\n",
+    "First of all, why a sum? \n",
+    "Since we decided to append the reward assigned in each state of the environment to a list, we end up with a pretty long one, especially in gargantuan mazes.\n",
+    "The events we defined and chose to record are `getCarrots` and `MountEvent` which respectively assign a reward of one point for picking up a carrot and 1000 points for mounting the steed.\n",
+    "\n",
+    "We discussed extensively about using an event that'd give off a reward of -1 for each step taken by the agent and came to the conclusion that it would be inadequate for our task: we wished to face multiple contingencies while taming the steed: fighting a monster in our path to a carrot or near the steed, finding all the carrots, getting the steed to drop the saddle if it ever wished to take it and many more. Because of this, using such a reward would have pushed us to solve the task *faster* instead of *better*, becoming a sort of toxic inductive bias on our work: we'd have been more focused on finding good heuristics to finish the level in time instead of solving the many problems we had to face. \n",
```

Comment:
remove double ":"

Reply:
update, this is full of ":" . why !!!!!!!!!!!

Reply:
replaced second : with , such as